src/Mod/Part/Gui/DlgProjectionOnSurface.cpp

```diff
@@ -178,7 +178,12 @@ DlgProjectionOnSurface::~DlgProjectionOnSurface()
   delete ui;
   for ( auto it : m_projectionSurfaceVec)
   {
-    higlight_object(it.partFeature, it.partName, false, 0);
+    try {
+      higlight_object(it.partFeature, it.partName, false, 0);
```

Comment:
should this be `highlight_object` ?

Reply:
Yes. That said, in the interest of keeping these static analysis PRs focused I am trying not to make any unrelated changes.